test/augmentation/test_augmentation.py

```diff
@@ -915,6 +915,28 @@ def test_same_on_batch(self, shape, erase_scale_range, aspect_ratio_range):
         print(f._params)
         assert (res[0] == res[1]).all()
 
+    @pytest.mark.parametrize("value", [99., (88., 66., 44.)])
+    def test_value(self, value, device, dtype):
+        torch.manual_seed(10)
+        input = torch.arange(10, device=device, dtype=dtype).reshape(2, 5).repeat(1, 3, 1, 1)
+        f = RandomErasing((0.3, 0.3), (0.3, 0.3), value=value, same_on_batch=False, p=1.)
```

Comment:
BTW, @shijianjian check this comment https://github.com/kornia/kornia/pull/684#discussion_r492263883
we should start trrying this for the functions that we want to overload